kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/jstl/KmeliaDisplayHelper.java

```diff
@@ -344,14 +324,6 @@ public static void displayWizardOperations(String wizardRow, String id,
   public static void displayOnNewOperations(KmeliaSessionController kmeliaScc,
       GraphicElementFactory gef, String action, JspWriter out) throws IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "gef". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1172)